src/components/Analysis/helpers.js

```diff
@@ -1,13 +1,14 @@
-import { modeToColor } from 'src/components/helpers'
+import { modeToColor, purposeToColor } from 'src/components/helpers'
 import { formatCO2 } from 'src/lib/trips'
 
-export const makeChartProps = (timeseriesSortedByModes, t) => {
-  const timeseriesData = Object.values(timeseriesSortedByModes).map(
-    modeValues => modeValues.totalCO2
+export const makeChartProps = (sortedTimeseries, type, t) => {
```

Comment:
nit: is that exported function tested and documented?

Reply:
non, on ne fait que retourner des objets directement utilisable par la lib. Il faut se référer à la doc de la lib si nécessaire.